public/sitemap.xml

```diff
@@ -1,19 +1,17 @@
-  <url loc="https://app.uniswap.org/pool/v2" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/pool" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/pools/v2/find" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/pools/v2" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/pools" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.7"/>
-  <url loc="https://app.uniswap.org/add/v2" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/add" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/increase" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/migrate/v2" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/nfts" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
-  <url loc="https://app.uniswap.org/nfts/profile" lastmod="2023-10-05T17:48:32.538Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="1"/>
+  <url loc="https://app.uniswap.org/tokens" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.8"/>
+  <url loc="https://app.uniswap.org/send" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/swap" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.9"/>
+  <url loc="https://app.uniswap.org/pool/v2/find" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/pool/v2" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/pool" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/pools/v2/find" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/pools/v2" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/pools" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.7"/>
+  <url loc="https://app.uniswap.org/add/v2" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/add" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/increase" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
+  <url loc="https://app.uniswap.org/migrate/v2" lastmod="2024-01-04T10:28:17.114Z" changefreq="weekly" priority="0.6"/>
```

Comment:
I'm not sure what changed in this file, but I think we should get rid of the hard-coded uniswap.org links.  Same for any other place `uniswap.org` shows up.  May require a bit of research to find out what exactly these represent, and whether it can be host independent or not.

Reply:
I removed /nfts and /profiles, and also last modification dates got changed